src/App.tsx

```diff
@@ -1,223 +1,100 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
-import { AuthContext } from './components/Auth/AuthContext';
+import React, { useContext, useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { AuthContext, AuthProvider } from './components/Auth/AuthContext';
+// eslint-disable-next-line max-len
+import { ErrorNotifications } from './components/ErrorNotifications/ErrorNotifications';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { TodoList } from './components/TodoList/TodoList';
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+import { Error } from './types/Error';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
-  const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<Status>(Status.All);
+  const [error, setError] = useState<Error>(Error.None);
+
+  const loadTodos = async () => {
+    if (!user) {
+      return;
+    }
+
+    setError(Error.None);
+
+    try {
+      const todosFromServer = await getTodos(user.id);
+
+      setTodos(todosFromServer);
+    } catch {
+      setError(Error.NoTodos);
+    }
+  };
 
   useEffect(() => {
-    // focus the element with `ref={newTodoField}`
-    if (newTodoField.current) {
-      newTodoField.current.focus();
+    loadTodos();
+  }, [user]);
+
+  const todosFilter = () => {
+    switch (status) {
+      case Status.Active:
+        return todos.filter(todo => !todo.completed);
+
+      case Status.Completed:
+        return todos.filter(todo => todo.completed);
+
+      case Status.All:
+      default:
+        return todos;
     }
-  }, []);
+  };
```

Comment:
put it in some type of useEffect

Reply:
ty for review) fixes are done, check pls :)